libopae/src/token_list.c

```diff
@@ -85,6 +86,11 @@ struct _fpga_token *token_add(const char *sysfspath, const char *devpath)
 		return NULL;
 	}
 
+	/* populate error list */
+	char errpath[SYSFS_PATH_MAX];
+	snprintf_s_s(errpath, SYSFS_PATH_MAX, "%s/errors", sysfspath);
```

Comment:
what about fme errors? those are found in the subdirectory `errors/fme-errors/errors` with their clear file being `errors/fme-errors/clear`

Reply:
Nevermind, I forgot this function recurses subdirectories